apps/ledger-live-desktop/tests/jestSetup.ts

```diff
@@ -13,8 +13,38 @@ jest.mock("@sentry/electron/renderer", () => ({
 
 jest.mock("src/sentry/install", () => ({
   init: jest.fn(),
+  setUser: jest.fn(),
+  captureException: jest.fn(),
+  addBreadcrumb: jest.fn(),
+  setTags: jest.fn(),
 }));
 
+class Worker {
+  url: string;
+  onmessage: (msg: string) => void;
+  constructor(stringUrl: string) {
+    this.url = stringUrl;
+    this.onmessage = jest.fn();
+  }
+  postMessage(msg: string): void {
+    this.onmessage(msg);
+  }
+}
+
+Object.defineProperty(window, "Worker", {
+  writable: true,
+  value: Worker,
+});
+
```

Comment:
Could you have a look if we can create a dedicated file for it, please? Because if we start adding more and more mocks it will be hard to read this file.